### DISCUSSION TITLE
Language owners

### DISCUSSION THREAD

**Discussion by jeroenterheerdt** [jeroenterheerdt/HAsmartirrigation#227] [Announcements]
English / Dutch: @jeroenterheerdt 
German: @gatonero 
Italian: @valerix85
French: @diyanei 
Norwegian: @hwikene
Spanish: @arivelli
Slovak: @misa1515

**Comment by gatonero** [comment]
@valerix85, @diyanei , @hwikene, @arivelli

Add translation to `/custom_components/smart_irrigation/frontend/localize/languages`f or "Automatic weather data pruning". There also some descriptions for the other fields which need translation.

**Comment by gatonero** [comment]
@jeroenterheerdt  who are the owners of es and no?

**Reply by jeroenterheerdt** [reply]
Norwegian: @hwikene

**Reply by jeroenterheerdt** [reply]
I'd recommend making a bug for each language and tag owner in it to make sure the localization files (both the core and the fronend) are in sync with EN

**Reply by jeroenterheerdt** [reply]
Spanish: @arivelli

**Reply by gatonero** [reply]
Some new issues!  ;-) 

I couldn't  assign the language owners therefore they are assigned to you by default.

**Reply by jeroenterheerdt** [reply]
yeah, in order to assign it to people they need to be collaborators on the repo.
